src/eos/eosEngine.js

```diff
@@ -190,7 +192,7 @@ export class EosEngine extends CurrencyEngine {
       networkFee: '0',
       parentNetworkFee: '0',
       ourReceiveAddresses,
-      signedTx: hex_data,
+      signedTx: 'n/a',
```

Comment:
Just set to `undefined`, or leave it out entirely. I'm pretty sure it's optional.